drawbot/json_types.py

```diff
@@ -1,9 +1,5 @@
 from typing import Dict, Any, List, Union
```

Comment:
Just put this into the `utils/` folder

Reply:
Sound fair enough, could move it when i'll be at home